doc/Support/Configuration.md

```diff
@@ -715,6 +715,93 @@ $config['ipmi']['type'][] = "open";
 
 [Distributed Poller](../Extensions/Distributed-Poller.md)
 
+### Wago data collection
```

Comment:
This should be under doc/Support/Device-Notes/

Reply:
Also, I still wouldn't mind a general approach.

Reply:
Well Tony, I'll move the doc at the right place. Thank you for the feedback.

Regarding the general approach, this is in my TODO list, but this needs more thinking to design a really generic form that could be used for this Wago device, as well as any new device. I'll start working on it as soon as I'll be back from holidays. 🌞